Sources/TuistCore/Models/LintingIssue.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
+import TuistSupport
 
 public struct LintingError: FatalError, Equatable {
     public var description: String = "Fatal linting issues found"
```

Comment:
wondering as we have got public init now, if you could turn turn these mutable properties into non-mutable ones.

Reply:
Agree. I'll make them non-mutable.